knownprojects_build/sql/_project_record_ids.sql

```diff
@@ -26,4 +26,25 @@ FROM(
     WHERE source NOT IN ('DOB', 'Neighborhood Study Rezoning Commitments', 'Future Neighborhood Studies')
 ) a
 WHERE id IS NOT NULL
-GROUP BY id;
\ No newline at end of file
+GROUP BY id;
+
+DROP TABLE IF EXISTS project_review;
```

Comment:
can we split this block out to another sql script?
also we should be exporting a shapefile for this table too